src/inject/index.ts

```diff
@@ -41,6 +43,25 @@ function onMessage({type, data}) {
     }
 }
 
+function getDataViaXHR() {
+    if (new RegExp(`(^|\\s|;)${chrome.runtime.id}=\\s*([-\\w]+)\\s*(;|$)`).test(document.cookie)) {
```

Comment:
`document.cookie` [should be accessed inside `try`](https://github.com/openstyles/stylus/commit/92fcb02a575e26139dbf0d827c48fb4dcaadfa98), otherwise it'll throw inside sandboxed frames.

Reply:
Thanks for letting me know! Implemented it in https://github.com/darkreader/darkreader/pull/3921/commits/842c87e2555d41bab3686a79ec72401744a1d83f